settings.gradle.kts

```diff
@@ -1,50 +1,17 @@
-import org.gradle.api.internal.FeaturePreviews
-
-pluginManagement {
-    plugins {
-        id("org.jetbrains.kotlin.jvm") version "1.8.10"
-        id("com.github.johnrengelman.shadow") version "8.1.0"
-    }
-}
```

Comment:
Why not keep common version declarations for plugins at the top level of the Gradle Build?

Wasn't that the model that idiomatic Gradle was pushing? Plugin versions weren't supposed to be declared in child build.gradle files IIRC.

Reply:
Gradle 8.1 supports `libs.plugins.X` in the plugins block. Using this mechanism seems to make the `pluginManagement` declaration unnecessary.